includes/admin/class-wc-rest-payments-payment-intents-controller.php

```diff
@@ -46,4 +77,286 @@ public function get_payment_intent( $request ) {
+							'description' => __( 'Billing details for the payment method.', 'woocommerce-payments' ),
+							'type'        => 'object',
+							'context'     => [ 'view' ],
+							'properties'  => [
+								'address' => [
+									'description' => __( 'Address associated with the billing details.', 'woocommerce-payments' ),
+									'type'        => 'object',
+									'context'     => [ 'view' ],
+									'properties'  => [
+										'city'        => [
+											'description' => __( 'City of the billing address.', 'woocommerce-payments' ),
+											'type'        => 'string',
+											'context'     => [ 'view' ],
+										],
+										'country'     => [
+											'description' => __( 'Country of the billing address.', 'woocommerce-payments' ),
+											'type'        => 'string',
+											'context'     => [ 'view' ],
+										],
+										'line1'       => [
+											'description' => __( 'Line 1 of the billing address.', 'woocommerce-payments' ),
+											'type'        => 'string',
+											'context'     => [ 'view' ],
+										],
+										'line2'       => [
+											'description' => __( 'Line 2 of the billing address.', 'woocommerce-payments' ),
+											'type'        => 'string',
+											'context'     => [ 'view' ],
+										],
+										'postal_code' => [
+											'description' => __( 'Postal code of the billing address.', 'woocommerce-payments' ),
+											'type'        => 'string',
+											'context'     => [ 'view' ],
+										],
+										'state'       => [
+											'description' => __( 'State of the billing address.', 'woocommerce-payments' ),
+											'type'        => 'string',
+											'context'     => [ 'view' ],
+										],
+									],
+								],
+								'email'   => [
+									'description' => __( 'Email associated with the billing details.', 'woocommerce-payments' ),
+									'type'        => 'string',
+									'format'      => 'email',
+									'context'     => [ 'view' ],
+								],
+								'name'    => [
+									'description' => __( 'Name associated with the billing details.', 'woocommerce-payments' ),
+									'type'        => 'string',
+									'context'     => [ 'view' ],
+								],
+								'phone'   => [
+									'description' => __( 'Phone number associated with the billing details.', 'woocommerce-payments' ),
+									'type'        => 'string',
+									'context'     => [ 'view' ],
+								],
+							],
+						],
+						'payment_method'         => [
+							'description' => 'The payment method associated with this charge.',
+							'type'        => 'string',
+							'context'     => [ 'view' ],
+						],
+						'application_fee_amount' => [
+							'description' => 'The application fee amount.',
+							'type'        => 'string',
+							'context'     => [ 'view' ],
+						],
+						'status'                 => [
+							'description' => 'The application fee amount.',
+							'type'        => 'string',
```

Comment:
Nitpick: Maybe this could be integer.

Reply:
MIstake, it was not an amount, corrected the description. It is a status string